solutions/instances/variables.tf

```
@@ -226,6 +226,27 @@ variable "profile_attachments" {
   default     = ["IBM Cloud Framework for Financial Services"]
 }
 
+variable "resource_groups_scope" {
+  type        = list(string)
+  description = "The resource group to scope the Security and Compliance Center profile attachments to. If left empty, it will default to being scoped to the current account id. This currently has a limitation of accepting a max number of one resource groups."
```

```suggestion
  description = "The resource group to associate with the Security and Compliance Center profile attachments. If not specified, the attachments are scoped to the current account ID. Only one resource group is allowed."
```

Do we need to provide a link to what this means and why a user cares? Like [Scanning your resources](https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-scan-resources)